docs/For Developers/Building NW.js.md

```diff
@@ -30,7 +30,7 @@ NW.js use same build tools and similar steps as Chromium. Read the instructions
 ```bash
 mkdir -p $HOME/nwjs
 cd $HOME/nwjs
-gclient config --name=src https://github.com/nwjs/chromium.src.git@origin/nw17
+gclient config --name=src https://github.com/nwjs/chromium.src.git@origin/nw65
```

Comment:
Could you instead create a Markdown variable for this so it's easier update in the future?

Reply:
Ok, I will spend some time to change it. Thanks

Reply:
Hi @ayushmxn It seems like there is no simple way to use Markdown variable to reflect the version. I was thinking to use Markdown variable to change according to the branch you checked out. I'm not good in Markdown. Do you have any suggestion?

Reply:
That's an interesting idea. I'm not sure if GitHub's Markdown supports that. I'll look into it.

Reply:
Did a quick search on the internet but was unable to find anything useful for this PR. For now we could just [create a Markdown variable](https://brianchildress.co/variables-in-markdown/)? If I do find a way to implement your idea, I can open a PR myself later.

Reply:
I tried the square bracket method, i think it doesnt work, the variable set wasnt show up in the actual place for replacement.
Maybe it is within a code block, so it doesnt detect a need to replace.

Reply:
Thanks a lot for taking the time to try and figure this out. I think it's fine if the md variable doesn't work since the priority is to update the docs.

cc @rogerwang

Reply:
The branch name doesn't matter here. `@` and the branch name can be removed.